### PR TITLE
[PX-399] Update demo project for sharing

### PR DESCRIPTION
**Pull request by kylemcnair**
For the purposes of sharing the demo repo, I'm working to make it usable for different audiences

Changes
* replace sources with seeds: this removes the requirement to have access to source tables in Snowflake
* revised readme
  * needs more work, but wanted to share progress 

Local `dbt build`
```
demo [kyle-px-399-updated-demo-environment] dbt build --full-refresh
20:50:08  Running with dbt=1.3.2
20:50:08  Found 4 models, 10 tests, 0 snapshots, 0 analyses, 303 macros, 0 operations, 5 seed files, 0 sources, 0 exposures, 0 metrics
20:50:08  
20:50:10  Concurrency: 10 threads (target='dev')
20:50:10  
20:50:10  1 of 19 START seed file DEV_KYLE_seed_data.feature_used ........................ [RUN]
20:50:10  2 of 19 START seed file DEV_KYLE_seed_data.org_created ......................... [RUN]
20:50:10  3 of 19 START seed file DEV_KYLE_seed_data.signed_in ........................... [RUN]
20:50:10  4 of 19 START seed file DEV_KYLE_seed_data.subscription_created ................ [RUN]
20:50:10  5 of 19 START seed file DEV_KYLE_seed_data.user_created ........................ [RUN]
20:50:13  3 of 19 OK loaded seed file DEV_KYLE_seed_data.signed_in ....................... [CREATE 157 in 2.45s]
20:50:13  2 of 19 OK loaded seed file DEV_KYLE_seed_data.org_created ..................... [CREATE 146 in 2.71s]
20:50:13  5 of 19 OK loaded seed file DEV_KYLE_seed_data.user_created .................... [CREATE 475 in 2.75s]
20:50:13  6 of 19 START test not_null_user_created_USER_ID ............................... [RUN]
20:50:13  7 of 19 START test unique_user_created_USER_ID ................................. [RUN]
20:50:13  4 of 19 OK loaded seed file DEV_KYLE_seed_data.subscription_created ............ [CREATE 53 in 2.77s]
20:50:14  1 of 19 OK loaded seed file DEV_KYLE_seed_data.feature_used .................... [CREATE 4561 in 3.11s]
20:50:14  6 of 19 PASS not_null_user_created_USER_ID ..................................... [PASS in 0.97s]
20:50:14  7 of 19 PASS unique_user_created_USER_ID ....................................... [PASS in 0.97s]
20:50:14  8 of 19 START sql table model DEV_KYLE.dim_orgs ................................ [RUN]
20:50:16  8 of 19 OK created sql table model DEV_KYLE.dim_orgs ........................... [SUCCESS 1 in 1.51s]
20:50:16  9 of 19 START test not_null_dim_orgs_org_id .................................... [RUN]
20:50:16  10 of 19 START test unique_dim_orgs_org_id ..................................... [RUN]
20:50:16  10 of 19 PASS unique_dim_orgs_org_id ........................................... [PASS in 0.78s]
20:50:16  9 of 19 PASS not_null_dim_orgs_org_id .......................................... [PASS in 0.79s]
20:50:16  11 of 19 START sql table model DEV_KYLE.fct_monthly_financials ................. [RUN]
20:50:16  12 of 19 START sql table model DEV_KYLE.sales_sync ............................. [RUN]
20:50:18  12 of 19 OK created sql table model DEV_KYLE.sales_sync ........................ [SUCCESS 1 in 1.18s]
20:50:18  13 of 19 START test not_null_sales_sync_org_id ................................. [RUN]
20:50:18  14 of 19 START test unique_sales_sync_org_id ................................... [RUN]
20:50:18  11 of 19 OK created sql table model DEV_KYLE.fct_monthly_financials ............ [SUCCESS 1 in 1.20s]
20:50:18  15 of 19 START test not_null_fct_monthly_financials_date_month ................. [RUN]
20:50:18  16 of 19 START test unique_fct_monthly_financials_date_month ................... [RUN]
20:50:18  14 of 19 PASS unique_sales_sync_org_id ......................................... [PASS in 0.70s]
20:50:19  13 of 19 PASS not_null_sales_sync_org_id ....................................... [PASS in 0.86s]
20:50:19  15 of 19 PASS not_null_fct_monthly_financials_date_month ....................... [PASS in 0.88s]
20:50:19  16 of 19 PASS unique_fct_monthly_financials_date_month ......................... [PASS in 0.90s]
20:50:19  17 of 19 START sql table model DEV_KYLE.fct_yearly_financials .................. [RUN]
20:50:20  17 of 19 OK created sql table model DEV_KYLE.fct_yearly_financials ............. [SUCCESS 1 in 1.43s]
20:50:20  18 of 19 START test not_null_fct_yearly_financials_date_year ................... [RUN]
20:50:20  19 of 19 START test unique_fct_yearly_financials_date_year ..................... [RUN]
20:50:21  19 of 19 PASS unique_fct_yearly_financials_date_year ........................... [PASS in 0.89s]
20:50:21  18 of 19 PASS not_null_fct_yearly_financials_date_year ......................... [PASS in 0.97s]
20:50:21  
20:50:21  Finished running 5 seeds, 10 tests, 4 table models in 0 hours 0 minutes and 13.30 seconds (13.30s).
20:50:21  
20:50:21  Completed successfully
20:50:21  
20:50:21  Done. PASS=19 WARN=0 ERROR=0 SKIP=0 TOTAL=19

### CODE REVIEW

**Review by leoebfolsom — APPROVED** [review]
This looks good to me but I haven't tested it--I'm approving so you're not blocked, and I can test it out later today if no one gets to it first.

**Comment by kylemcnair** [comment]
I'm gonna merge this as an MVP. I will iterate on some cleaner examples and on the readme docs.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+
+5. Ensure your profile is setup correctly from the command line:
+```bash
+$ dbt debug
+```
+
+6. Run the models:
+```bash
+$ dbt build
+```
```
Comment (williebsweet) [inline README.md:37]: Do we need to instruct users to target dev and prod here? i.e. build with --target prod, then make a change and build with --target dev?

Comment (williebsweet) [reply README.md:37]: cc: @kylemcnair

Comment (kylemcnair) [reply README.md:37]: good point, I'll add that

**Pull request merged**